### DISCUSSION TITLE
Welcome to docker-unifi-rpi Discussions!

### DISCUSSION THREAD

**Discussion by ryansch** [ryansch/docker-unifi-rpi#104] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ryansch** [comment]
I made this! 😉

**Comment by brianjbeach** [comment]
I just wanted to thank you for maintaining this image. I have been using it for a few year now. It is very much appreciated.

**Reply by ryansch** [reply]
You're welcome!
